README.md

```diff
@@ -319,6 +329,9 @@ sqlboiler -b goose_migrations postgres
 go test ./models
 ```
 
+*Note: No `mysqldump` or `pg_dump` equivalent for Microsoft SQL Server, so generated tests must be supplemented by `tables_schema.sql` with `CREATE TABLE ...` queries*
```

Comment:
After doing some research I came to the conclusion that everything that is present on the network is oriented to Windows GUI / CLI / Powershell (uses .NET assemblies) or abandoned a long time ago (3-7 years).

We can generate a schema from the code, but the result will not be exactly accurate - there will be no, for example, triggers that can affect the tests results.